49 adicionar portais de notcias

YanYamim opened the pull request:
<h2>O que foi feito até agora<h2>

<h3>Frontend</h3>
<ul>
  <li>Botão de cadastro de portais</li>
  <li>Campos com nome, url, checkbox se está ativo e seletor de tags para relacionar o portal à uma tag</li>
  <li>Botão de salvar e cancelar</li>
</ul>

<h3>Backend</h3>
<ul>
  <li>Controller para os portais com o método POST, além do record de cadastro de portais</li>
  <li>Interface dos portais</li>
</ul>

<h3>O que pode melhorar</h3>
<p>Por algum motivo não está cadastrando os dados no banco quando clica no botão de salvar (Não está aparecendo erros no painel de inspecionar e nem erros que quebram o código)</p>

Yan-Yamim committed:
✨ Criação da função de cadastrar portais

Yan-Yamim committed:
🎨 Implementação do seletor de tags para o Portal e correção de erros

c137santos commented on frontend/src/views/PortaisView.vue:
```diff
@@ -1,13 +1,30 @@
+    },
+
+    methods: {
+      adicionarPortal(novoPortal) {
```
Cade a chamada para o backend para salvar no banco?

YanYamim replied:
Não é a linha de baixo: this.portais.push(novoPortal) ?

Commit:
🩹 Corrigindo nome da classe para o css funcionar

o nome da classe estava cadastrar-porta, corrigi para cadastro-porta e o css então foi aplicado. Além de retirar alguns comentários no código

Yan-Yamim committed:
🐛 Portais agora estão sendo cadastrados no banco com sucesso

Yan-Yamim committed:
🚧 adição do tagService, repository para cadastro do Id de tag e portal no banco

Commit:
🚧 Refatorando para comportar service

Recebendo um erro de nullpoint agora na query. Avançamos mais

Yan-Yamim committed:
🚧 Refatorando para corrigir erros

Commit:
✨ Enviando tagId,  TagPortalrepository e controler

Corrigindo o envio de tagId para backend, a forma do controller tratar as informações e o TagPortalRepository
